client/src/featureform/serving_test.py

```diff
@@ -6,6 +6,15 @@
 print(fpf)
 # # fpf = client.features([("SepalLength", "centimeters")], {"CustomerID": "1"})
 # # Run features through model
-training_set = client.training_set("join", "v1")
+training_set = client.training_set("jjoin", "v1")
```

Comment:
why jjoin

Reply:
Was testing error handling. Ignore this branch for now

Reply:
Does this need to be changed back?